.github/workflows/build.yml

```
@@ -1,6 +1,6 @@
 name: Build
 
-on: [push, pull_request]
+on: [ push, pull_request ]
```

Let's avoid unnecessary changes

```suggestion
on: [push, pull_request]
```

---

Reverted.